recipes/hybridcontents/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hybridcontents" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: af8e3f6886533748761e41bd12690a9c4854e2a12baae2d49f5b72258f3138e4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - notebook >=4.0
+    - six >=1.9.0
+    - requests >=2.7.0
+  run:
+    - python
+    - notebook >=4.0
+    - six >=1.9.0
+    - requests >=2.7.0
+
+test:
+  imports:
+    - hybridcontents
+  requires:
+    - pytest
+    - mock
+    - nose
+  source_files:
+    - hybridcontents/tests/*.py
+  commands:
+    - pytest hybridcontents/tests/
+
+about:
+  home: https://github.com/viaduct-ai/hybridcontents
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "Hybrid Content Manager"
+
+  # The remaining entries in this section are optional, but recommended.
+  description: "A ContentsManager wrapper for using multiple ContentsManager in Jupyter"
```

Comment:
You can leave this description, or use something else from the homepage, e.g.: "The HybridContentManager was originally created by Quantopian as part of pgcontents; however, the usage of HybridContentsManager was restricted to the compatibility requirements of pgcontents. These restrictions included postgres dependencies and no support for the latest notebook version (>6). At Viaduct we used pgcontents exclusively for the HybridContentsManager and wanted to extend its functionality, so we created this fork hybridcontents." etc.

Reply:
Updated